packages/admin/tests/test-helper.js

```diff
@@ -0,0 +1,8 @@
+import Application from '../app';
+import config from '../config/environment';
+import { setApplication } from '@ember/test-helpers';
+import { start } from 'ember-qunit';
+
+setApplication(Application.create(config.APP));
+
```

Comment:
Update this file to be typescript
https://github.com/yahoo/navi/blob/3558bc7ce116ecd3cee7f2fc965b3721a127cb0c/packages/data/tests/test-helper.ts#L1-L12